### ISSUE TITLE
Undocumented requirement for mitmproxy not greater than 0.16

### ISSUE THREAD

**Issue by etogithub** [jgoldshlag/padherder_proxy#36]
You will get either ProxyConfig "host" key unknown or libmproxy not found, depending on how you are trying to hack around the problem.

One solution:   pip install  mitmproxy==0.16
Another is to update proxy.ProxyConfig usage if you have a newer version of mitmproxy ... it looks like it doesn't take "port/host" anymore.

**Comment by jgoldshlag** [comment]
Yeah, I changed the docs to mention that, and I need to look at what else has changed and probably upgrade. Going to leave this open to remind me to do so.
